src/Augurk/Augurk.csproj

```diff
@@ -1,216 +1,31 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\packages\Microsoft.Web.LibraryManager.Build.1.0.163\build\Microsoft.Web.LibraryManager.Build.props" Condition="Exists('..\packages\Microsoft.Web.LibraryManager.Build.1.0.163\build\Microsoft.Web.LibraryManager.Build.props')" />
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>
-    </ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{13364A29-2A72-4F65-AE13-28B270801EB3}</ProjectGuid>
-    <ProjectTypeGuids>{349c5851-65df-11da-9384-00065b846f21};{fae04ec0-301f-11d3-bf4b-00c04f79efbc}</ProjectTypeGuids>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Augurk</RootNamespace>
-    <AssemblyName>Augurk</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <UseIISExpress>false</UseIISExpress>
-    <IISExpressSSLPort />
-    <IISExpressAnonymousAuthentication />
-    <IISExpressWindowsAuthentication />
-    <IISExpressUseClassicPipelineMode />
-    <Prefer32Bit>false</Prefer32Bit>
-    <UseGlobalApplicationHostFile />
-    <TargetFrameworkProfile />
-    <NuGetPackageImportStamp>
-    </NuGetPackageImportStamp>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
We'll have to think about the versions of .NET Core we're going to support. 2.1 is the LTS release so we're good for now, but we'll need to think if we're going to use .NET 2.2 when it comes out.

Reply:
Probably best to stay in sync with LTS?

Reply:
I think so, but we'll also need to see what RavenDB is going to do. Even RavenDB embedded is currently tied to a specific .NET Core 2.1 version (with 4.1.2 being tied to 2.1.5 and 4.1.3 being tied to 2.1.6). Staying at LTS is probably the right thing to do for now though.